.github/ISSUE_TEMPLATE/bugfix.yml

```
@@ -0,0 +1,392 @@
+name: Bug Fix
+description: Report a bug for fixing using Claude Code and AI Review-First design
+title: "[BUG] "
+labels: ["bug", "claude-code"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Fix Template
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)